src/components/form/examples/help-form-schema.ts

```diff
@@ -1,6 +1,16 @@
 import { FormSchema } from '@limetech/lime-elements';
 
-export const schema: FormSchema = {
+type Race = 'search' | 'recommendation' | 'ads' | 'work' | 'workshop';
```

Comment:
This raised my eyebrow a bit… 😆

I had to track it down, and this was clearly a copy-and-paste error made [here](https://github.com/Lundalogik/lime-elements/pull/2622/files#diff-ffd4c8bf682a861e2eec3afe1191504fa1625a4e6c5638b5414857a981617f3dR45). However, this PR makes the mistake a whole lot more visible, so it would be nice to correct it now.

Reply:
Nice find 👍 I just continued with the copy/paste 😂
But I can fix it. I will just do a force push

Reply:
Is now fixed